src/app/_components/Card.tsx

```diff
@@ -165,7 +165,7 @@ export function Card({
           title
         )}
 
-        <div className="mb-10 mt-3">
+        <div className={clsx('mt-3', cta && 'mb-10')}>
```

Comment:
not sure if we need mb-10? but we can leave that for refactor

Reply:
We need it for the Cards with CTA.